AutorestSwiftTest/AutoRestSwaggerBatByteTest.swift

```diff
@@ -0,0 +1,124 @@
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import AutoRestSwaggerBatByte
+import AzureCore
+import XCTest
+
+class AutoRestSwaggerBatByteTest: XCTestCase {
+    var client: AutoRestSwaggerBatByteClient!
+
+    override func setUpWithError() throws {
+        client = try AutoRestSwaggerBatByteClient(
+            authPolicy: AnonymousAccessPolicy(),
+            withOptions: AutoRestSwaggerBatByteClientOptions()
+        )
+    }
+
+    func test_getnull200() throws {
+        let expectation = XCTestExpectation(description: "Call byte.getNull")
+
+        client.byte.getNull { result, httpResponse in
+            switch result {
+            case let .success(data):
+                XCTAssertEqual(data.count, 0)
+                XCTAssertEqual(httpResponse?.statusCode, 200)
+            case let .failure(error):
+                let details = errorDetails(for: error, withResponse: httpResponse)
+                XCTFail("Call getNull failed. error=\(details)")
+            }
+            expectation.fulfill()
+        }
+        wait(for: [expectation], timeout: 5.0)
+    }
+
+    func test_getEmpty200() throws {
+        let expectation = XCTestExpectation(description: "Call byte.getEmpty")
+
+        client.byte.getEmpty { result, httpResponse in
+            switch result {
+            case let .success(data):
+                XCTAssertEqual(String(decoding: data, as: UTF8.self), "\"\"")
+                XCTAssertEqual(httpResponse?.statusCode, 200)
+            case let .failure(error):
+                let details = errorDetails(for: error, withResponse: httpResponse)
+                XCTFail("Call getEmpty failed. error=\(details)")
+            }
+            expectation.fulfill()
+        }
+        wait(for: [expectation], timeout: 5.0)
+    }
+
+    func test_getNonAscii200() throws {
+        let expectation = XCTestExpectation(description: "Call byte.getNonAscii")
+
+        client.byte.getNonAscii { result, httpResponse in
+            switch result {
+            case let .success(data):
+                XCTAssertEqual(String(decoding: data, as: UTF8.self), "\"//79/Pv6+fj39g==\"")
+                XCTAssertEqual(httpResponse?.statusCode, 200)
+            case let .failure(error):
+                let details = errorDetails(for: error, withResponse: httpResponse)
+                XCTFail("Call getNonAscii failed. error=\(details)")
+            }
+            expectation.fulfill()
+        }
+        wait(for: [expectation], timeout: 5.0)
+    }
+
+    func test_getInvalid() throws {
+        let expectation = XCTestExpectation(description: "Call byte.getInvalid")
+
+        client.byte.getInvalid { result, httpResponse in
+            switch result {
+            case let .success(data):
+                XCTAssertEqual(String(decoding: data, as: UTF8.self), "\"::::SWAGGER::::\"")
```

Comment:
Why is the "get invalid" case expecting success?

Reply:
For the test case we get, `\"::::SWAGGER::::\"`  in http response body. 

In python test case, it expects a deserialize error. 

https://github.com/Azure/autorest.python/blob/autorestv3/test/vanilla/AcceptanceTests/test_byte.py

For swift, we are using Data object for byteArray, and we just since http response body is already in Data object, we just pass the Data back to the caller.

```
if [
                    200
                ].contains(statusCode) {
                    dispatchQueue.async {
                        completionHandler(.success(data), httpResponse)
                    }
                }
```

That is why we don't get any deserialize error. Plus the string ``\"::::SWAGGER::::\"` is a valid ASCII string, I am not sure why python test code expects a deserialize error.